crypto/aes/aes_core.c

```diff
@@ -50,7 +50,637 @@
 #include <openssl/aes.h>
 #include "aes_local.h"
 
-#ifndef AES_ASM
+#if !defined(OPENSSL_NO_AES_CONST_TIME) && !defined(AES_ASM)
```

Comment:
There is a CHANGES.md entry in 1.1.1g..

Is this still valid????

   Enable with: ./config no-asm -DOPENSSL_AES_CONST_TIME
   Disable with: ./config no-asm -DOPENSSL_NO_AES_CONST_TIME
   At this time this feature is by default disabled.
   It will be enabled by default in 3.0.

Reply:
Yes, if this PR gets merged, it will be as it has been written there.